README.md

```diff
@@ -6,22 +6,24 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/slint-ui/slint)](https://api.reuse.software/info/github.com/slint-ui/slint)
 [![Discussions](https://img.shields.io/github/discussions/slint-ui/slint)](https://github.com/slint-ui/slint/discussions)
 
+[#MadeWithSlint Applications Gallery](https://madewithslint.com/)
```

Comment:
I'm not sure about this. 
The README is an important page to get right since that's where everyone goes. And IMHO it should start with the description of what is Slint.

I think references to madewithslint belongs in our examples/demo sections.

Any reason you think it should be at the top?

Maybe if you really want it at the top, it could be in a badge at the top?   ![MadeWithSlint](https://img.shields.io/badge/%23MadeWithSlint-fff?logo=slint)  (to be polished)